src/platform/Linux/dbus/bluez/DbusBluez.xml

```diff
@@ -183,7 +183,7 @@
     <property name="ServiceData" type="a{sv}" access="read"/>
     <property name="Data" type="a{yay}" access="read"/>
     <property name="Discoverable" type="b" access="read"/>
-    <property name="DiscoverableTimeout" type="q" access="read"/>
+    <!-- <property name="DiscoverableTimeout" type="q" access="read"/> -->
```

Comment:
is this commenting-out needed given that the property is read?
A comment should explain why this was manually commented out.

Reply:
It is needed to be commented-out, because otherwise it will be present in the `org.freedesktop.DBus.Properties.GetAll()` D-Bus call. Since we can set this property to any value, BlueZ will complain that the value is not set, and will not register advertising. The same thing is with other commented-out properties on that interface.

Reply:
Maybe the thing to note in the comment here is that this XML definition is the basis for the "skeleton" for an object we create, rather than an object we consume. At least this wasn't obvious to me until looking at the code in more detail.

Reply:
What threw me off is that it says "access read" and the bug explains how we should not set DiscoverableTimeout ... but if access is read, why is setting it a thing.

This is what I would like explained in a comment: saying this exists, however it is commented out because it affects us ... in what way. The way I read it is "we remove a read-access property because we should not write it" and if I read it that way I get confused.

Reply:
Ok, I'll update the comment so it will be more clear what "set" means in that context.

Reply:
@andy31415 Could you please check whether the comments are easier to understand now?